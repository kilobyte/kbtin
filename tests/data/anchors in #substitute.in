#sub foo FOO
#sub ^bar BAR
#sub ^baz$ BAZ
#sub quux$ QUUX
#run a {sh -c 'for x in foobarbazquux foo bar baz quux xfoo fbar xbaz xquux;do echo "$x";done'}
