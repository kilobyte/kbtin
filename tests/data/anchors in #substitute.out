FOObarbazQUUX
FOO
BAR
BAZ
QUUX
xFOO
fbar
xbaz
xQUUX

#SESSION 'a' DIED.
#THERE'S NO ACTIVE SESSION NOW.
